meta-lmp-bsp/recipes-kernel/linux/linux-lmp-toradex-imx_git.bb

```diff
@@ -2,7 +2,7 @@ include recipes-kernel/linux/kmeta-linux-lmp-5.4.y.inc
 
 LINUX_VERSION ?= "5.4.129"
 KBRANCH = "toradex_5.4-2.3.x-imx"
-SRCREV_machine = "09dedcf8e8301c81290944fec7f46498a6057fb6"
+SRCREV_machine = "828ec42a6a32004a199f8ee4893fecdc0b15ae92"
```

Comment:
Technically this is the HEAD commit for Toradex: https://git.toradex.com/cgit/linux-toradex.git/commit/?h=toradex_5.4-2.3.x-imx&id=f28f1c4529ddede4f033e4d55fad00c9df3afe10 (1 higher than what you have)

Reply:
Sure, let me update that, it seems it got pushed today.

Reply:
Updated.